src/Regulator.cpp

```diff
@@ -463,9 +450,6 @@ UNDERRUN : {
 void Regulator::processPacket(bool glitch)
 {
     double tmp = 0.0;
-    if ((glitch) && (mFPPratioDenominator > 1)) {
```

Comment:
Not sure what this was for, but it seems to fix the sound issues when there is 5% loss